translations/fi.po

```diff
@@ -635,3 +635,51 @@ msgstr "Tilaa uutisten RSS-syöte valintojesi perusteella"
 msgctxt "React search clear selection label"
 msgid "Clear @label selection"
 msgstr "Tyhjennä @label valinnat"
+
```

Comment:
Some of these can be moved here https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/tree/main/translations